### ISSUE TITLE
Update build.gradle

### ISSUE THREAD

**Issue by nhatth219** [472647301/react-native-kline#12]
Thank you for amazing lib for candle stick.
I tried integrating to my react-native app but got an issue that relevant to outdate libs from build.gralde (android)
Could you please take a look, thanks @472647301 
<img width="734" alt="Screen Shot 2023-05-09 at 11 18 25" src="https://user-images.githubusercontent.com/114040146/236992984-baa2b159-e283-4643-aa01-c0c5cfb9a433.png">

**Comment by lap888** [comment]
相同的问题 猫是不是已经不维护了 0.72有问题

**Comment by ekoprtok** [comment]
any update for this ? @nhatth219

**Comment by ekoprtok** [comment]
> 相同的问题 猫是不是已经不维护了 0.72有问题

same, any update for this?

**Comment by ekoprtok** [comment]
> 相同的问题 猫是不是已经不维护了 0.72有问题

same, any update for this?

**Comment by volodya-korol** [comment]
@ekoprtok did you manage to solve the error?

**Comment by ekoprtok** [comment]
yes, we need to update build.gradle inside this package and change 
apply plugin: 'maven'
to
apply plugin: 'maven-publish'
@volodya-korol

**Comment by zaidqureshi95** [comment]
@ekoprtok I'm getting this issue on line 23 
```
A problem occurred configuring project ':react-native-kline'.
> Configuration with name 'compile' not found.
```

**Comment by itsnyx** [comment]
@zaidqureshi95  same issue here
did you find any fix?

**Comment by jibo001** [comment]
same problem

**Comment by 472647301** [comment]
We are migrating to the new architecture recently, and it is expected to take another week.

**Comment by itsnyx** [comment]
@472647301  thanks for the update

Can you please upgrade the charting libraries as well ? 
the scroll was not smooth on IOS and had problems 
also had some performance issues on android as well 

thanks
